.gitignore

```diff
@@ -35,3 +35,5 @@ helper.txt
 *.iml
 .vscode
 .idea
+
+book
```

Comment:
Is that where the book will be built?

Reply:
No, this is the old book folder name. Now it is `clippy_book`.
It is fixed for next push.